backend/app/services/auth_service.py

```
@@ -110,7 +112,7 @@ async def login(self, login_data: UserLogin):
                 )
                 return JSONResponse(
                     status_code=HTTPStatus.NOT_FOUND,
-                    content={"message": "User not found"},
+                    content={"message": USER_NOT_FOUND_MSG},
```

Ruff (0.8.2)

Undefined name `USER_NOT_FOUND_MSG`

(F821)